### PR TITLE
feat(participants-pane) updates

### PR DESCRIPTION
**Pull request by Calinteodor**
1.  Fixed react native community slider to work on both android and ios
2. Removed InviteButton from native menus
3. Fixed buttons spacing in native OverflowMenu
4. Participant context menu details are shown only for remote participants

### CODE REVIEW

File: android/sdk/build.gradle [hunk 1]
```diff
@@ -75,6 +75,7 @@ dependencies {
     implementation project(':react-native-webrtc')
     implementation project(':react-native-webview')
     implementation project(':react-native-splash-screen')
+    implementation project(':react-native-slider')
```
Comment (saghul) [inline android/sdk/build.gradle:78]: Can you please sort this alphabetically? Looks like we also missed it with splash-screen.

File: android/settings.gradle [hunk 2]
```diff
@@ -28,4 +28,6 @@ project(':react-native-svg').projectDir = new File(rootProject.projectDir, 	'../
 include ':react-native-webrtc'
 project(':react-native-webrtc').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-webrtc/android')
 include ':react-native-webview'
-project(':react-native-webview').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-webview/android')
\ No newline at end of file
+project(':react-native-webview').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-webview/android')
+include ':react-native-slider'
```
Comment (saghul) [inline android/settings.gradle:32]: Ditto alpha sort.

File: react/features/toolbox/components/native/Toolbox.js [hunk 3]
```diff
@@ -101,7 +100,6 @@ function Toolbox(props: Props) {
                         styles = { buttonStylesBorderless }
                         toggledStyles = { backgroundToggledStyle } />)}
                 {additionalButtons.has('tileview') && <TileViewButton styles = { buttonStylesBorderless } />}
-                {additionalButtons.has('invite') && <InviteButton styles = { buttonStylesBorderless } />}
```
Comment (saghul) [inline react/features/toolbox/components/native/Toolbox.js:104]: Add the people button here instead.

**Review by saghul — COMMENTED** [review]
Left some comments, PTAL!

File: android/sdk/build.gradle (hunk 1 above)
Comment (Calinteodor) [reply android/sdk/build.gradle]: Fixed.

File: react/features/toolbox/components/native/Toolbox.js (hunk 3 above)
Comment (Calinteodor) [reply react/features/toolbox/components/native/Toolbox.js:104]: Replaced with ParticipantsPaneButton

**Review by saghul — APPROVED** [review]

**Pull request merged**